src/register/gintenr.rs

```diff
@@ -17,15 +17,20 @@ pub fn read() -> usize {
 
 #[inline]
 pub unsafe fn write(bits: usize) {
-    asm!("csrs 0x800, {}", in(reg) bits);
```

Comment:
this `csrs` expands to `csrrs x0, 0x800, bits`, which applies the bits as a bitmask instead of "writing"